recipes-config/systemd-resource-control/slices/slice.connectivity.conf

```diff
@@ -0,0 +1,3 @@
+[Unit]
```

Comment:
Rename slice.connectivity.conf to connectivity.slice if you agree that we shall not use the override feature for slice unit configuration (only for service unit configuration / adaptations)